test_unstructured/metrics/test_evaluate.py

```diff
@@ -125,3 +140,63 @@ def test_text_extraction_wrong_type():
         measure_text_extraction_accuracy(
             output_dir=output_dir, source_dir=source_dir, export_dir=export_dir, output_type="wrong"
         )
+
+
+@pytest.mark.skipif(is_in_docker, reason="Skipping this test in Docker container")
```

Comment:
This reason just restates the presence of the decorator. I can tell it does this from the `@..skipif(is_in_docker,..)` part. What might be more useful is a statement of _why_ this test is skipped in docker.

Reply:
I will create another ticket to deal with this specifically :) Thank you so much

Reply:
https://unstructured-ai.atlassian.net/browse/CORE-3980